src/game/interactables_2/note_particle.c

```diff
@@ -26,7 +26,7 @@ typedef struct {
 } Sprite_NoteParticle; /* size: 0x4C */
 
 /* {anim, variant, tileCount, shouldAllocTileMem(?), (unk | priority)} */
-static const u16 gUnknown_080E0140[4][5] = {
+static const ALIGNED(4) u16 gUnknown_080E0140[4][5] = {
```

Comment:
Is this still needed?

Reply:
In my head, this shows it should be a struct array? As I think struct arrays align 4?